css/tideline.less

```diff
@@ -164,8 +164,8 @@ svg {
   }
 }
 
-// messages
-.d3-rect-message {
+// messages, timechange
+.d3-rect-message, .d3-rect-timechange {
```

Comment:
Isn't the `d3-rect-message` the highlight box that pops up on click? If so, I don't think it's being used for the time changes since they are just hover interaction.